commit 8dbea8f26466f605961b66e8c3ae2da5f88a50a9
Author: Volker Krause <vkrause@kde.org>
Date:   Sat Aug 5 18:35:10 2017 +0200

    Only require Qt5X11Extras when we actually need it
    
    Reviewers: #frameworks
    
    Tags: #frameworks
    
    Differential Revision: https://phabricator.kde.org/D7154

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1111d73..e3c89d8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -33,7 +33,7 @@ set(REQUIRED_QT_VERSION 5.6.0)
 find_package(Qt5 ${REQUIRED_QT_VERSION} CONFIG REQUIRED Widgets)
 
 if(NOT WIN32 AND NOT APPLE)
-    find_package(Qt5 ${REQUIRED_QT_VERSION} CONFIG REQUIRED X11Extras DBus)
+    find_package(Qt5 ${REQUIRED_QT_VERSION} CONFIG REQUIRED DBus)
 endif()
 
 if(APPLE)
@@ -46,6 +46,7 @@ if(NOT APPLE)
 endif()
 
 if(X11_FOUND)
+    find_package(Qt5 ${REQUIRED_QT_VERSION} CONFIG REQUIRED X11Extras)
     find_package(XCB COMPONENTS XCB SYNC)
     find_package(X11_XCB)
     set(HAVE_X11 TRUE)
